### DISCUSSION TITLE
Get firebase sdk version in gradle file

### DISCUSSION THREAD

**Discussion by svenlombaert** [invertase/react-native-firebase#5569] [Q&A]
Hi there!

We had to add some custom code for using appboy-sdk with firebase messaging. All seems to work but this means I had to add these lines to my `app/build.gradle` file:

```
implementation platform("com.google.firebase:firebase-bom:28.1.0")
implementation "com.google.firebase:firebase-messaging"
```

As you can see I just hardcoded the SDK version, but is there a way to read out the SDK version that's specified in the messaging package?

**Comment by mikehardy** [comment]
Indeed there is, you might like: https://github.com/invertase/react-native-firebase/blob/39aa46f404f571f05ea16b23932f25325582f3bb/packages/messaging/android/build.gradle#L23-L33
